examples/todo-list/README.md

```diff
@@ -0,0 +1,116 @@
+# @loopback/example-todo-list
+
+This is an extended tutorial from `@loopback/example-todo`.
+
+## Overview
+
+This tutorial demonstrates how to create a set of APIs for models that are
+related to another.
+
+![todo-tutorial-overview](./imgs/todo-list-overview.png)
+
+## Setup
+
+You'll need to make sure you have some things installed:
+
+- [Node.js](https://nodejs.org/en/) at v8.x or greater
+
+Additionally, this tutorial assumes that you are comfortable with certain
+technologies, languages and concepts.
+
+- JavaScript (ES6)
```

Comment:
I don't know how I feel about this entire setup since the target audience is people who are familiar with the framework and have likely done the first tutorial ... so should have Node installed.

I think a good way to structure this section might be to do the following:
- Say if they are continuing from `todo-tutorial` then to jump to the next step (link to it). 
- If they are not continuing ... then they should make sure they have Node installed, CLI installed. I don't think we need to say we assume they need to know JavaScript (ES6) or REST ... and technically that's incorrect since the tutorial is technically in TS.  And lastly show them how to catch up -- `lb4 example todo`.